spec/granite_orm/transactions/import_spec.cr

```diff
@@ -0,0 +1,59 @@
+require "../../spec_helper"
+
+{% for adapter in GraniteExample::ADAPTERS %}
+module {{adapter.capitalize.id}}
+  describe "{{ adapter.id }} .import" do
+    it "should import 3 new objects" do
+      to_import = [
+        Parent.new(name: "ImportParent1"),
+        Parent.new(name: "ImportParent2"),
+        Parent.new(name: "ImportParent3"),
+      ]
+      Parent.import(to_import)
+      Parent.all("WHERE name LIKE ?", ["Import%"]).size.should eq 3
+		end
```

Comment:
Is it me or is the indentation is all strange in this file?

Reply:
Yea dunno, is fine on my local so not sure what is up with that.  I'll re indent and see if that fixes it next commit.

Reply:
Yup, fixed :p